developers/weaviate/model-providers/ollama/embeddings.md

```diff
@@ -103,7 +103,13 @@ As this integration connects to a local Ollama container, no additional credenti
 
 </Tabs>
 
-The [default model](#available-models) is used if no model is specified.
+If you don't specify an embedding model, the collection uses the [default model](#available-models).
+
+import APIEndpoint from '/_includes/integrations/ollama/api-endpoint.mdx';
+
+<APIEndpoint/>
+
+For more information on configuring a vectorizer, see [Specify a vectorizer](/developers/weaviate/manage-data/collections#specify-a-vectorizer).
```

Comment:
(See above)

Reply:
All of those redirect the reader away from the focus on the current page. In the case of Ollama the redirection is worse because the redirected page lacks the critical API config information

Reply:
I don't agree that in-line links take the reader away from the focus. 

I have heard from users that they do like in-line links. Also, they are also more concise. If you do insist on this change could you please make the corresponding change on all pages  here so we do not forget? Thanks